src/transformStateWithClones.js

```diff
@@ -7,7 +7,31 @@
  * @return {Object[]}
  */
 function transformStateWithClones(state, actions) {
-  // write code here
+  const result = [];
+  let backupObj = Object.assign({}, state);
```

Comment:
spread operator would make it even more concise 🙂 
```suggestion
  let backupObj = { ...state };
```

Reply:
Might as well :D applying